### PR TITLE
Fix logging of where application is bound

### PR DESCRIPTION
**Pull request by hut8**
Addresses https://github.com/gobuffalo/buffalo/issues/2248

This adds a method to a public interface, which isn't ideal and may technically be a breaking change. I'm open to suggestions about that.

### CODE REVIEW

**Comment by hut8** [comment]
Log message is now:

```
 msg="starting server of type *servers.TLS at 0.0.0.0:443"
```

**Review by sio4 — CHANGES_REQUESTED** [review]
Thank you for the PR!
I wrote some comments so please kindly consider them.

Briefly, I would like to minimize the changes to the interface since we are now planning a next version and don't want to introduce breaking changes while I fully agreed that the wrong information should be fixed.

File: server.go [hunk 1]
```diff
@@ -22,10 +22,6 @@ import (
 func (a *App) Serve(srvs ...servers.Server) error {
 	var wg sync.WaitGroup
 
-	// FIXME: this information is not correct.
-	// It needs to be fixed as we support multiple servers.
-	a.Logger.Infof("starting application at http://%s", a.Options.Addr)
```
[the remark this answers is not in the document]
Comment (sio4) [reply server.go:27]: but this is not a critical :-)

File: servers/simple.go [hunk 2]
```diff
@@ -17,6 +17,11 @@ func (s *Simple) SetAddr(addr string) {
 	}
 }
 
+// Addr gets the HTTP server address
+func (s *Simple) Addr() string {
```
[the remark this answers is not in the document]
Comment (hut8) [reply servers/simple.go:21]: Great solution 👍🏻 That resolves my main concern

**Comment by hut8** [comment]
```
DEBU[2022-04-26T20:02:55Z] starting application
INFO[2022-04-26T20:02:55Z] starting simple server on 0.0.0.0:3000
```
That's the new output.

**Review by sio4 — APPROVED** [review]
Thanks! LGTM

**Comment by sio4** [comment]
Hi @paganotoni,
Could you please take a look at this PR and merge it? This PR improved the startup message.
(what happen with the codeclimate? :thinking: )

**Comment by hut8** [comment]
Absolutely. My mistake! Thanks for correcting me on that. I'll get this
done today or tomorrow

**Review by paganotoni — APPROVED** [review]
LGTM!

**Comment by paganotoni** [comment]
@hut8 thanks for putting together this one. I attempted to rebase it to development but seems it has some conflicts. Can you please take care of those. Thanks again!

**Comment by hut8** [comment]
@paganotoni I think I took care of it 😄

**Pull request merged**